features/layout/blocks/articles/list-block.js

```diff
@@ -38,44 +38,59 @@ const arrangeListInBlocks = articles =>
     []
   );
 
-export const ArticlesList = ({ articles }) => {
+export const ArticlesList = ({ articles, imageLoading = 'eager' }) => {
   if (articles.length === 1) {
     const { articleId } = articles[0];
     return (
       <FeaturedBlock
         // This is a workaround in order to use FeaturedBlock as it is.
         block={{ frozen: true, articleId }}
         data={{ articles: { [articleId]: articles[0] } }}
+        imageLoading={imageLoading}
       />
     );
   }
 
   if (articles.length === 3) {
     // If articles' list contains exactly 3 articles, we treat it in a special way:
     // we do not want 3 articles to be rendered in a same way as *3 first articles* are rendered in generic composition.
-    return <TagPageBlockCD articles={articles} layout={LAYOUT_BY_LEVEL.C} />;
+    return (
+      <TagPageBlockCD articles={articles} layout={LAYOUT_BY_LEVEL.C} imageLoading={imageLoading} />
+    );
   }
 
   const blocks = arrangeListInBlocks(articles);
 
   return blocks.map((block, index) => {
     const levelName = PAGE_LEVEL_ORDER[index % PAGE_LEVEL_ORDER.length];
     const Block = BLOCK_BY_LEVEL[levelName];
-    // eslint-disable-next-line react/no-array-index-key
-    return <Block key={index} articles={block} layout={LAYOUT_BY_LEVEL[levelName]} />;
+    return (
+      <Block
+        // eslint-disable-next-line react/no-array-index-key
+        key={index}
+        articles={block}
+        layout={LAYOUT_BY_LEVEL[levelName]}
+        imageLoading={index === 0 ? imageLoading : 'lazy'}
```

Comment:
I'd say `index === 1` also should inherit `imageLoading`, probably `index === 2` also.

Reply:
no, I checked: first block is enough